concepts/0167-consent-lifecycle/README.md

```diff
@@ -0,0 +1,668 @@
+private data processed. This RFC will list the technology changes
+required to implement GDPR. Note the RFC focuses on preserving the
+processing agreement associated with the consent rather than consent
+dialogue. Consent dialogue is the presentation of a meaningful consent
+without in a consent.
+
+Hyperledger Indy and Aries provide the perfect frameworks for managing
+personal data specially personal identifiable information (PII). When
+necessary data is restricted to protect the identity of the individual
+or data subject. Currently the privacy policy that is agreed to when
+signing up for a new service dictates how personal data is processed and
+for which purpose. Currently there is no clear technology to hold a
+company accountable for the privacy policy. By using blockchain and the
+consent receipt, accountability of a privacy policy can be reached. The
+consent is not limited to a single data controller (or institution) and
+data subject (or individual) but to a series of institutions that
+process the data from the original data subject. The beauty of the
+proposal in this RFC is that the accountability is extended to ALL
+parties using the data subject's personal data. When the data subject
+withdraws consent so is the consent receipt agreement.
+
+GDPR lacks specifics regarding how technology should be or can be used
+to enforce obligations. This RFC provides a viable alternative with the
+mechanisms to bring accountability and at the same time protecting
+personal data.
+
+Overview
+========
+
+There are three key areas that need to be in place:
+
+1.  Schema bases/overlays
+
+2.  Consent Lifecycle
+
+3.  Wallet
+
+The schema bases/overlay describes a standard approach to data capture
+that separates raw schema building blocks from additional semantic
+layers such as data entry business logic and constraints, knowledge
+about data sensitivity, and so forth \[refer to [[overlay
+RFC]](https://github.com/hyperledger/aries-rfcs/tree/master/concepts/0013-overlays)
+for details\]. The consent lifecycle covers the consent receipt
+certificate, proof request and revocation. The wallet is where all data
+is stored which requires a high level of security and control by
+individual or institution. This RFC will cover the consent lifecycle.
+
+The concepts section help better understand the RFC in GDPR terms. There
+is an attempt to align with the vocabulary defined in specification
+[[W3C Data Privacy Vocabulary]](https://www.w3.org/ns/dpv).
+
+To best understand the implementation guidelines several use cases with
+a reference implementation will serve as a demo \[To Be Done: add
+reference to demo in indy-sdk\]. The following illustration is an
+overview of the key actors.
+
+![](media/consent_overview.png)
+
+Concepts
+--------
+
+These are some concepts that are important to understand when reviewing
+this RFC.
+
+**Secondary Data Controller**: The terms data subject and data
```

Comment:
I think it would be helpful to put this in terms of SSI as well.  How do issuer, holder and verifier align with the "** data controller" terminology?  Does it vary by use case?

Reply:
I strongly agree with this suggestion.

Reply:
Added a reference to SSI. Plan to use terms more frequently when explaining the use cases in more detail.